tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/DefaultDeliveryDateControllerTests.cs

```diff
@@ -92,9 +92,10 @@ public static void Constructor_VerifyGuardClauses()
             [Frozen] Mock<IDefaultDeliveryDateValidator> validator,
             DefaultDeliveryDateController controller)
         {
-            service.Setup(o => o.GetDefaultDeliveryOrder(callOffId, catalogueItemId)).ReturnsAsync(order);
-            service.Setup(o => o.SetDefaultDeliveryDate(callOffId, catalogueItemId, defaultDeliveryDate.DeliveryDate)).Callback(() =>
+            service.Setup(o => o.GetOrder(callOffId, catalogueItemId)).ReturnsAsync(order);
+            service.Setup(o => o.SetDefaultDeliveryDate(callOffId, catalogueItemId, defaultDeliveryDate.DeliveryDate.Value)).Callback(() =>
             {
+                // ReSharper disable once PossibleInvalidOperationException (covered by model validation)
                 order.SetDefaultDeliveryDate(catalogueItemId, defaultDeliveryDate.DeliveryDate.Value);
```

Comment:
Rather than introduce the ReSharper comment here I think we can just use the null-forgiving operator (`!`) as AutoFixture  should supply us with a non-null delivery date value.

Reply:
Used null-forgiving operator `(!)` instead of ReSharper comment